client/src/app/management/components/meeting-list/meeting-list.component.html

```diff
@@ -58,6 +58,16 @@ <h2>
         </div>
     </div>
 
+    <div *pblNgridCellDef="'organisation_tags'; row as meeting" class="cell-slot fill">
+        <div *ngIf="meeting.organisation_tags?.length">
+            <mat-chip-list>
+                <os-chip *ngFor="let tag of meeting.organisation_tags" [color]="tag.color">
```

Comment:
Just a suggestion: Make a `os-tags` component taking in a list (or a single one) of tags, so it is even easier to use.

Reply:
Hmm, I don't think, that this is a good pattern. Let us discuss this on monday.